dvc/command/remote.py

```diff
@@ -116,7 +116,7 @@ def add_parser(subparsers, parent_parser):
 
     fix_subparsers(remote_subparsers)
 
-    REMOTE_ADD_HELP = "Add remote."
+    REMOTE_ADD_HELP = "Add a new data remote."
```

Comment:
Are we sure we need `data` here? Won't it be confusing in regards to `models` and such?

Reply:
Sorry to repeat the same answer haha but just matching docs here (https://dvc.org/doc/command-reference/remote/add).

For consistency, we use the term "data remote" in remote subcommands cmd refs (not in other docs though). This is kind of arbitrary, I agree on that. Would you prefer a different standard? Happy to update everywhere.